packages/design-system/src/components/TextField/TextField.test.tsx

```diff
@@ -20,11 +20,46 @@ describe('TextField', function () {
   });
 
   it('renders with a label mask', () => {
-    expect(renderTextField({ labelMask: DATE_MASK }).asFragment()).toMatchSnapshot();
+    const { container } = renderTextField({ labelMask: DATE_MASK });
+
+    const label = container.querySelector('.ds-c-label');
+    expect(label).toBeInTheDocument();
```

Comment:
For some of these labels, we could also check that their `for` attribute matches the `input.id` so we know they're properly linked. That's not something that our VRTs tell us. It's possible that that's covered by our a11y tests though.